video/video.ino

```diff
@@ -102,7 +102,7 @@ void loop() {
 		if (processTerminal()) {
 			continue;
 		}
-		if (millis() - cursorTime > CURSOR_PHASE) {
+		if (cursorFlashing && (millis() - cursorTime > cursorFlashRate)) {

```

Comment:
this logic change ensures that we'll only flash the cursor if `cursorFlashing` is set, and now makes use of a variable flash rate